.github/workflows/os-check.yml

```diff
@@ -32,5 +32,5 @@ jobs:
       - name: cargo check
         uses: actions-rs/cargo@v1
         with:
-          command: check
+          command: test
```

Comment:
This doesn't seem right — this was intentionally `check` as the name of the job implies.

Reply:
Right, l will change the name of the job to match what it does then.

IMO it is necessary to run cargo-test on the github action so that it doesn't accidentally break on MacOS.

Reply:
Done!

Reply:
Actually I've been developing on MacBook Air 2020 for some time and it's been working quite good.
With Docker Desktop, I can run all the integration tests without any problem.

Reply:
Right, but that doesn't work on GitHub actions